src/FixedDataTableStore.js

```diff
@@ -12,9 +12,16 @@
 'use strict';
 
 import reducers from 'reducers';
-import { createStore } from 'redux';
+import { configureStore } from '@reduxjs/toolkit';
 
-const store = createStore(reducers);
+const store = configureStore({
+  reducer: reducers,
+  middleware: getDefaultMiddleware => getDefaultMiddleware({
+    serializableCheck: false,
+    // Todo: Have to disable immutableCheck because state has circular JSON somewhere in it. Need to investigate it.
+    immutableCheck: false
```

Comment:
State is a circular data structure, hence immutable check will always result in an infinite loop. Have to disable this as well.

Reply:
It might be worth ticketing out fixing this as well.

Reply:
Created a ticket for it
https://jira.schrodinger.com/browse/SS-32510